docs/content/en/blog/releases/v0.42.0.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Release v0.42.0"
+linkTitle: "Release v0.42.0"
+date: 2023-02-27
+description: >
+ Release v0.42.0
+---
+
+## Changes since v0.41.4
+
+### New Features
+
+* Add last use of api key on webui ([#4124](https://github.com/pipe-cd/pipecd/pull/4124))
+* Add pipectl deployment list ([#4182](https://github.com/pipe-cd/pipecd/pull/4182))
```

Comment:
I think it's more clear to move to Notable changes as well as list applications by labels. wdyt?

Reply:
for sure, updated 👌